consultation_analyser/support_console/ingest.py

```diff
@@ -63,6 +65,7 @@ def get_consultation_codes() -> list[dict]:
 
         # Get unique consultation folders
         consultation_codes = set()
+        print("trying", settings.AWS_BUCKET_NAME)
```

Comment:
Assume this needs to come out!

Reply:
quite right [2c86d16](https://github.com/i-dot-ai/consult/pull/675/commits/2c86d165a3d2a33ec9c1a223019b1b0597cf9f4a)!